src/components/box/box.spec.tsx

```diff
@@ -71,4 +81,13 @@ describe("Box", () => {
       assertStyleMatch({ boxSizing }, wrapper);
     }
   );
+
+  it.each<Gap>([...numericalGapValues, "20px", "20%"])(
+    "has styles applied when gap is set to %s",
+    (gapValue) => {
+      const wrapper = mount(<Box display="flex" gap={gapValue} />);
+      const gap = boxConfig.gap(gapValue);
+      assertStyleMatch({ gap }, wrapper);
```

Comment:
I'm not sure whether this is a useful test. It's basically testing "are we calling `boxConfig.gap(gapValue)` and applying that to the CSS `gap` property?".

I guess that's still a useful thing to check but I'd want to also include tests for `boxConfig.gap` to make sure that's doing what it should.

Reply:
I'll change it so `const gap = typeof gapValue === number ? `${gapValue * 8}px` : gapValue` but don't feel there's any more value in writing tests for the config function past that